install.sh

```diff
@@ -1815,6 +1234,15 @@ IPAM_DRIVER_EQUINIX_SCRIPTS="src/ipamm_mad/remotes/equinix/register_address_rang
                             src/ipamm_mad/remotes/equinix/get_address \
                             src/ipamm_mad/remotes/equinix/free_address"
 
+#-------------------------------------------------------------------------------
+# IPAM Scaleway drivers to be installed under $REMOTES_LOCATION/ipam
+#-------------------------------------------------------------------------------
+IPAM_DRIVER_SCALEWAY_SCRIPTS="src/ipamm_mad/remotes/scaleway/register_address_range \
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

IPAM_DRIVER_SCALEWAY_SCRIPTS appears unused. Verify use (or export if used externally).